MODULE.bazel

```diff
@@ -33,6 +33,7 @@ bazel_dep(name = "rules_testing", version = "0.6.0")
 bazel_dep(name = "googletest", version = "1.14.0.bcr.1", repo_name = "com_google_googletest")
 bazel_dep(name = "with_cfg.bzl", version = "0.2.4")
 bazel_dep(name = "abseil-cpp", version = "20230802.1")
+bazel_dep(name = "nlohmann_json", version = "3.11.3")
```

Comment:
Is there a more standard/minimal dependency to parse JSON?

We'd like to have the minimal count dependencies used by Bazel. If adding this module is the only way, it will require more scrutiny.